src/calendar/index.ts

```diff
@@ -158,16 +177,16 @@ export class CalendarBase<P extends CalendarProperties = CalendarProperties> ext
 		this.invalidate();
 	}
 
-	private _onDateClick(date: number, disabled: boolean) {
+	private _onDateClick(date: number, disabled: boolean, currentMonth: boolean) {
+		if (disabled) {
```

Comment:
this seem's a strange place do to this, I'd have expected the `calendarCell` to simply not call the `onClick` if it's disabled / invalid